### ISSUE TITLE
iOS: FrameworkElement.ActualHeight and .ActualWidth not working correctly

### ISSUE THREAD

**Issue by DierkDroth** [unoplatform/uno#4285]
## Current behavior

Download attached, set breakpoint in Main.xaml.cs 32 and start on iOS simulator.
[UnoSample.zip](https://github.com/unoplatform/uno/files/5389747/UnoSample.zip)

## Expected behavior

.ActualHeight/.ActualWidth should have correct values as with UWP

## Workaround

None I know of

## Environment

Last UNO packages

Affected platform(s):

- [x] iOS  (haven's tested others)
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [x ] Visual Studio 2019 (version: latest)
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

**Comment by jeromelaban** [comment]
Thanks for the report. This issue is related to the ordering of the Loaded and Unloaded events (https://github.com/unoplatform/uno/issues/2895)

**Comment by jeromelaban** [comment]
For this issue, until 2895 is fixed, it's best to read the values of `ActualWidth`/`ActualHeight` when invoked from the `LayoutUpdated` event.

**Comment by DierkDroth** [comment]
Thanks for letting me know, @jeromelaban. I'll look into...

**Comment by DierkDroth** [comment]
@jeromelaban FYI initial impression: this could help in my custom project. I'll have to run more test...

**Comment by DierkDroth** [comment]
@jeromelaban when ripping through my code and putting in the .LayoutUpdated event handlers I noticed that there are various cases where .LayoutUpdated is constantly fired (I have a breakpoints there which are hit constantly) although the UI is static (and even in the background).

Is that expected?

**Comment by jeromelaban** [comment]
Yes, this handler is invoked when the layout is updated (generally when ArrangeOverride is called). 

Note that ActualWidth and ActualHeight are generally non-performant properties because they are not observable. It's generally best to find other ways to implement what you are trying to do.

**Comment by DierkDroth** [comment]
>> Yes, this handler is invoked when the layout is updated (generally when ArrangeOverride is called).
Understand, but should it be constantly fired although the rendering is done, and the UI is static?

>> It's generally best to find other ways to implement what you are trying to do.
Ok, what else should I be using to get the 'actual/real' width/height of components?

**Comment by jeromelaban** [comment]
You can make your own panel/control for this, and measure children properly.

**Comment by DierkDroth** [comment]
@jeromelaban that's the point. 

I have use cases where this is a Canvas which I'm using for custom rendering, but for doing that I needed the actual Width/Height

**Comment by MartinZikmund** [comment]
This works correctly for Skia renderer targets
